backend/sirius-components-collaborative-diagrams/src/main/java/org/eclipse/sirius/components/collaborative/diagrams/export/svg/ImageRegistry.java

```diff
@@ -144,11 +144,6 @@ private String updateSvgContent(byte[] content) {
 
         String cleanSvgString = new String(content).replaceAll(xmlDeclaration, ""); //$NON-NLS-1$
 
-        String svgWidthPattern = "width=\".*?\""; //$NON-NLS-1$
-        cleanSvgString = cleanSvgString.replaceFirst(svgWidthPattern, ""); //$NON-NLS-1$
-        String svgHeightPattern = "height=\".*?\""; //$NON-NLS-1$
-        cleanSvgString = cleanSvgString.replaceFirst(svgHeightPattern, ""); //$NON-NLS-1$
-
```

Comment:
This part has fixed https://github.com/eclipse-sirius/sirius-components/issues/1106 in addtion to the label icon display
I am sorry I can not remember the use case where this code was useful